syntax = "proto2";
package spider;

message producer_register_req {
	required int32 id = 1;
}
